test.e2e/test-server.js

```diff
@@ -161,6 +149,27 @@ masterApp.listen(port, function() {
     runAndExit(process.argv[2], process.argv.slice(3));
 });
 
+function generateService(generator, lbApp, apiUrl, opts) {
+  var servicesScript;
+  if (typeof opts.includeSchema !== 'undefined' ||
```

Comment:
In Loopback, we use simpler `opts.includeSchema !== undefined`. Please simplify here and on the lines below too.